src/nvim/lua/treesitter.c

```diff
@@ -158,7 +158,8 @@ int tslua_add_language(lua_State *L)
 
   uv_lib_t lib;
   if (uv_dlopen(path, &lib)) {
-    snprintf((char *)IObuff, IOSIZE, "Failed to load parser: uv_dlopen: %s",
+    snprintf((char *)IObuff, IOSIZE, "Failed to load parser for lang %s: uv_dlopen: %s",
```

Comment:
maybe use `language` here to the user facing error message?

Reply:
agreed. I have some trouble running the tests locally (I use host deps, which have a too old luv) but will fix that asap.